### PR TITLE
Fix doc building command

### PR DESCRIPTION
**Pull request by aefhm**
**Why**

Fix doc building.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -54,7 +54,7 @@ lint: $(lint-targets)
 docs-targets := docs-api
 
 docs-api:
-	@npx --no redoc-cli build api/spec/v1.yaml -o index.html
```
Comment (aefhm) [inline Makefile:57]: Unless we meant `--no-install`?

Comment (ennsharma) [reply Makefile:57]: I may have 😅

**Review by ennsharma — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (aefhm) [reply Makefile:57]: Oh, I'm on an older version of `npx`.

Comment (aefhm) [reply Makefile:57]: Hmm, I think we still need the install.

**Pull request merged**